cli/cmd/plugin/ui/web/tanzu-ui/src/shared/components/management-cluster-settings-step/ManagementClusterSettings.tsx

```diff
@@ -53,20 +58,27 @@ const nodeProfiles = [
         value: 'PRODUCTION_READY',
     },
 ];
-function ManagementClusterSettings(props: Partial<MCSettings>) {
-    const { handleValueChange, currentStep, deploy, defaultData, message } = props;
+
+function ManagementClusterSettings<T>(props: Partial<MCSettings<T>>) {
+    const { handleValueChange, currentStep, deploy, defaultData, message, getImageMethod, clusterName } = props;
     const {
         register,
         formState: { errors },
     } = useForm<FormInputs>();
     const [selectedProfile, setSelectedProfile] = useState('SINGLE_NODE');
+    const [images, setImages] = useState<T[]>([]);
+    const setImageParameters = (image) => {
+        if (handleValueChange) {
+            handleValueChange(INPUT_CHANGE, 'IMAGE_INFO', image, currentStep, errors);
+        }
+    };
     const handleNodeProfileChange = (event: ChangeEvent<HTMLSelectElement>) => {
         setSelectedProfile(event.target.value);
         if (handleValueChange) {
             handleValueChange(INPUT_CHANGE, 'NODE_PROFILE', event.target.value, currentStep, errors);
         }
     };
-
+    const { awsState } = useContext(AwsStore);
```

Comment:
He, it looks like your changes to add the os image step got mixed into this PR. You may want to remove these changes to keep the tooltip PR clean.

Reply:
Yes sir. My bad. Gonna create a new PR then